oidc-service-impl/src/main/java/io/mosip/esignet/services/ConsentHelperService.java

```diff
@@ -192,7 +181,7 @@ private String generateSignedObject(OIDCTransaction transaction, ConsentDetail c
             jwsObject = new JWSObject(jwsHeader.toBase64URL(), Base64URL.encode(claimsSet.toJSONObject().toJSONString())
                     ,Base64URL.encode(signature) );
         } catch (ParseException e) {
-
+            log.error(e.getLocalizedMessage());
```

Comment:
it is always better to log errors as below, so that it prints the complete stacktrace.
log.error("generate signed object failed ", e);





[11:42](https://mosip-team.slack.com/archives/D04TD6S21BN/p1687068758162349)
log.error("operation invoked ", e);